dags/probe_scraper.py

```diff
@@ -478,4 +376,4 @@ def choose_branch(self, context):
         dag=dag,
     )
 
-    lookml_generator_prod >> glean_dictionary_netlify_build
+    trigger_looker >> glean_dictionary_netlify_build
```

Comment:
Does `TriggerDagRunOperator` succeed immediately after triggering or does it wait for the run status?

Reply:
There is a `wait_for_completion` parameter which would wait for the DAG run. I'll add it.
Although Glean dictionary doesn't really depend on anything that is in Looker

Reply:
Found it https://airflow.apache.org/docs/apache-airflow/stable/_api/airflow/operators/trigger_dagrun/index.html

There's a `wait_for_completion` param that defaults to false.  Seems like we want it to be true?  Assuming glean dictionary is actually dependent on lookml, I'm not sure

Reply:
I added it to keep it consistent with the current state